src/cr-logging.eventstore/cr-logging.eventstore.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
If we're packaging straight from the project files we should make sure they have useful information (author set to Cognisant, etc.) Update the URLs for the project to point to GitHub, the license file, and a logo. And add some useful tags.